hack/prom-rule-ci/rule-spec-dumper.go

```diff
@@ -5,15 +5,17 @@ import (
 	"fmt"
 	"os"
 
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/rules"
+	hcoRules "github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/hyperconverged/rules"
+	observabilityRules "github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/observability/rules"
 )
 
 func verifyArgs() error {
 	numOfArgs := len(os.Args[1:])
-	if numOfArgs != 1 {
-		return fmt.Errorf("expected exactly 1 argument, got: %d", numOfArgs)
+	if numOfArgs != 2 {
+		return fmt.Errorf("expected exactly 2 arguments, got: %d", numOfArgs)
```

Comment:
this is an internal tool that should run from automation, but still, this error message does not help the user to understand what to put in these two parameters. Please consider more detailed error message.

Reply:
updated